cmd/dep/glide_config.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"io/ioutil"
+	"path"
+	"path/filepath"
+
+	"github.com/go-yaml/yaml"
+	"github.com/golang/dep"
+	"github.com/golang/dep/internal/gps"
+	"github.com/pkg/errors"
+)
+
+type glideFiles struct {
+	yaml    glideYaml
+	lock    *glideLock
+	loggers *dep.Loggers
+}
+
+func newGlideFiles(loggers *dep.Loggers) *glideFiles {
+	return &glideFiles{loggers: loggers}
+}
+
+type glideYaml struct {
+	Name        string         `yaml:"package"`
+	Ignores     []string       `yaml:"ignore"`
+	ExcludeDirs []string       `yaml:"excludeDirs"`
+	Imports     []glidePackage `yaml:"import"`
+	TestImports []glidePackage `yaml:"testImport"`
+}
+
+type glideLock struct {
+	Imports     []glidePackage `yaml:"import"`
+	TestImports []glidePackage `yaml:"testImport"`
+}
+
+type glidePackage struct {
+	Name       string `yaml:"package"`
+	Reference  string `yaml:"version"`
+	Repository string `yaml:"repo"`
+
+	// Unsupported fields that we will warn if used
+	Subpackages []string `yaml:"subpackages"`
+	OS          string   `yaml:"os"`
```

Comment:
Just noting here that we _may_ be able to deal with these eventually - #291 - and if we warn on seeing these, we should probably include a link to that issue.